pkg/asset/installconfig/openstack/validate.go

```diff
@@ -65,7 +65,14 @@ func ValidateForProvisioning(ic *types.InstallConfig) error {
 	return nil
 }
 
-func defaultOpenStackMachinePoolPlatform() openstack.MachinePool {
+func defaultMasterOpenStackMachinePoolPlatform() openstack.MachinePool {
+	return openstack.MachinePool{
+		ServerGroupPolicy: "soft-anti-affinity",
```

Comment:
Adding this here is not going to have any effect. This is only used for configuring the `Machine` and `MachineSets`. And `Machine` does not support the field.